### PR TITLE
Work

### PR DESCRIPTION
**Pull request by tingamapuro04**
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

### CODE REVIEW

**Comment by tingamapuro04** [comment]
I have generally acted upon linter errors.

**Review by koomekevo — APPROVED** [review]

**Review by koomekevo — CHANGES_REQUESTED** [review]
Hi @tingamapuro04 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly implement the works section as suggested in the below screenshot

tingacorrect

The below screenshot is your design

tingaworksectionwrong


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

I have added a wrap to the first and third cards in the class s-container.

File: index.html [hunk 1]
```diff
@@ -35,14 +35,236 @@ <h1> <strong>I’m Mariane
...
+                    <ul>
+                        <li class="html">html</li>
+                        <li class="css">css</li>
+                        <li class="java">javaScript</li>
+                    </ul>
+                </div>
+                <div class="Submission">
+                    <button type="submit">See Project</button>
+                </div>
+            </section>
```
Comment (vic778) [inline index.html]: `Figma template`
Screenshot from 2022-04-30 09-42-37
`your work`

Screenshot from 2022-04-30 09-42-30

- [ ] Kindly You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hello @tingamapuro04 

Good job so far! I know you have worked hard on making this project.  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [x] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tingamapuro04** [comment]
I am working on the work branch where i have introduced three more containers to the mobile skeleton. The first container describe my work and skills. In this container i have added four sub-sections with each sub-section having an image at the top and then followed by the body and then a button. I have used grid to place these four sub-sections.
In the second container I have introduced contains information about self generally . In the last container contains contact information and i have also included a form for user input.
I have continued to work the files such as index.html and nina.css

I have added a wrap to the first and third cards in the class s-container.

I have changed the font-sizes in the the class s-container.

**Review by KDlamini — APPROVED** [review]
### Hi @tingamapuro04 🙋‍♀️,  

Great work! 💯 
You have done a great job implementing the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**